src/vs/platform/notification/common/notification.ts

```diff
@@ -320,8 +320,10 @@ export interface INotificationService {
 	 * Shows a prompt in the notification area with the provided choices. The prompt
 	 * is non-modal. If you want to show a modal dialog instead, use `IDialogService`.
 	 *
-	 * @param onCancel will be called if the user closed the notification without picking
-	 * any of the provided choices.
+	 * @param severity the severity of the notification. Either `Info`, `Warning` or `Error`.
```

Comment:
Not related directly with the implementation but it was outdated documentation of a method that was needed to complete feature.